oblt-cli/run/action.yml

```diff
@@ -24,7 +24,7 @@ runs:
         slack-channel: ${{ inputs.slack-channel }}
         username: ${{ inputs.username }}
     - name: run oblt-cli
-      run: oblt-cli ${COMMAND} --verbose --disable-banner
+      run: oblt-cli ${{ inputs.command }} --verbose --disable-banner
```

Comment:
what's the reason? as far as I see `env.COMMAND` is defined in L31

Reply:
woops. this was me testing stuff.

Reply:
https://github.com/elastic/oblt-actions/pull/13/commits/464246c2f93326d133ae07712628cca8080f56fd